src/ingest_validation_tools/table-schemas/assays/ims-v2.yaml

```diff
@@ -0,0 +1,132 @@
+doc_url: https://portal.hubmapconsortium.org/docs/assays/maldi-ims
+fields:
+# include: ../includes/vA/all_assays.yaml
+- name: assay_category
+  constraints:
+    enum:
+      - mass_spectrometry_imaging # TODO: This is an internal ID; would prefer not to change.
+- name: assay_type
+  constraints:
+    enum:
+      - MALDI-IMS
+      - SIMS-IMS
+      # TODO: "NanoDESI" is already a value under the nano schema: It can't be in both.
```

Comment:
```suggestion
      - nanoDESI
```

Reply:
nanoDESI has now been integrated into IMS along with the other mass spec imaging assays. Archive nanoDESI schema.